src/reactor.cc

```diff
@@ -413,9 +413,9 @@ bool Reactor::Discharge() {
 
   std::stringstream ss;
   ss << npop << " assemblies";
-  Record("DISCHARGE", ss.str());
-
+  Record("DISCHARGE", ss.str());  
   spent.Push(core.PopN(npop));
+  cyclus::toolkit::RecordTimeSeries<double>("supply"+fuel_outcommods[0], this, spent.quantity());
```

Comment:
Yeah, i agree. But there is only one spent buffer. Do you have any suggestions? Because all other notion of looping through outcommods only occurs in trading functions.